### ISSUE TITLE
UI/UX - Inconsistent button behaviour 

### ISSUE THREAD

**Issue by techieanant** [dashpay/dash-website#431]
**Problem**: All buttons on the website are <a> tags styled as buttons except for the "download-detected-platform-button" on https://www.dash.org/get-dash/ and "Join" button on https://www.dash.org/contact/. Due to these buttons being <button> and not <a>, the mouse hover behaviour is inconsistent as <a> have a pointer on hover whereas <button> do not. 

**Solution**: Change <button> tags to <a> on line 30 in file /contact/index.html and on line 23 in file /_includes/hero/get-dash.html.

**Comment by chuckwilliams37** [comment]
Good catch, @techieanant!  I know this is negatively affecting a minority of our users who may be using screenreaders, or other assistive devices to navigate the web. Also - as you've pointed out, the styling is behaving unexpectedly.

@tungfa This is another one for the "medium" priority items for the web team.  Thanks again, @techieanant !
